config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -626,6 +626,8 @@ periodics:
         label:triage/accepted
         -label:priority/important-soon
         -label:priority/critical-urgent
+        -label:"help wanted"
+        -label:lifecycle/frozen
```

Comment:
This is repeated three times because we currently have different durations depending on the `priority/` label

I don't think the priority label is well-regularized or understood currently, for now I think we should make this change to all of the jobs, and revisit the priority concept later.